src/babylon.assetContainer.ts

```diff
@@ -0,0 +1,120 @@
+module BABYLON {
+    export class AssetContainer {

```

Comment:
I changed this from SceneAssetContainer to AssetContainer. I can change this if anyone thinks the old name is better or has a better suggestion.

Reply:
looks good to me